pcdshub/lcls-twincat-motion
# MNT: CI, Pre-commit, Example Cleanup

- Update CI to latest best practices
- Add pre-commit hooks
- Move Example PLC project to its own repo, for ease of use and to make it a more representative example: https://github.com/pcdshub/lcls-twincat-motion-example
- Set target PLC to local instead of the motion ioc testing PLC
- Update `typhos` test screens for recent renaming
- Remove unused `Config` directory. This had some experimental utilities for hooking up to NC and CoE values. This can be revived via the magic of version control when it is needed again. I'll make sure there is an issue about this so it is not forgotten.

closes #63
